### ISSUE TITLE
Static analyzer warning about variable used before initialization in TinyGltfImporter

### ISSUE THREAD

**Issue by jonesmz** [mosra/magnum-plugins#104]
```
/home/jonesmz/osp-magnum/3rdparty/magnum-plugins/src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp: In member function ‘virtual void Magnum::Trade::TinyGltfImporter::doOpenData(Corrade::Containers::ArrayView<const char>)’:
/home/jonesmz/osp-magnum/3rdparty/magnum-plugins/src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp:272:27: warning: ‘callbacks.tinygltf::FsCallbacks::WriteWholeFile’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  272 |     tinygltf::FsCallbacks callbacks;
      |                           ^~~~~~~~~
```

**Comment by mosra** [comment]
Yep, I started getting this warning after an upgrade to GCC 10 as well. The cause of it was harmless though (it was for a `WriteWholeFile` callback, which isn't used) nevertheless it's good to have a warning-free build. 

Fixed in 310228fc88e76fbf4a9c3a775f3b4d01400695ed, sorry for the extreme delay.

**Comment by jonesmz** [comment]
Thanks!
